app/frontend/src/theme.ts

```diff
@@ -23,6 +23,15 @@ const themeOptions: ThemeOptions = {
       fontSize: "28px",
       marginTop: "8px",
     },
+    h4: {
+      fontSize: "24px",
+    },
+    h5: {
+      fontSize: "20px",
+    },
```

Comment:
```suggestion
    h4: {
      fontSize: 24,
    },
    h5: {
      fontSize: 20
    },
```
I think they default to `px` anyway? Maybe it doesn't matter for now but I think using px values are bad for accessibility, since the font won't zoom if the user views this at like 200% or something. Something worth thinking about in future.

Reply:
I even just tested it: but both `rem` and `px` zoom as well. I still think that a unit that is (eventually) related to rem is the best.

Reply:
I think that's because it zooms the whole page. What if you change the default font size in your browser settings? I think then only `rem` would zoom correctly.